frontend/lib/public/views/Overview/General/page.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * This file is part of the ALICE Electronic Logbook v2, also known as Jiskefet.
+ * Copyright (C) 2020  Stichting Hogeschool van Amsterdam
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import { h } from '/js/src/index.js';
+import filters from '../../../components/Filters/index.js';
+import { rowData, rowHeader } from '../../../components/Table/index.js';
+
+export default model => [overviewScreen(model)];
+
+/**
+ * Table row header
+ * @param {object} model
+ * @return {vnode}
+ */
+const overviewScreen = model => {
+  const headers = model.overview.getHeaders();
+  const data = model.overview.getTableData();
+  const tags = model.overview.getTagCounts();
+
+  return h('.w-75.flex-row', [
```

Comment:
Any reason for not using 100% width?

Reply:
With 100% width the table will overflow out of the screen, because tables in WebUI are hardcoded to also have 100% width. For that reason we can't add in 100% width to this container using the current layout.